mods/voidmulti.sc2mod/base.sc2data/gamedata/behaviordata.xml

```diff
@@ -33,7 +42,16 @@
         <Start value="Footprint4x4CreepSourceGrown"/>
         <Birth value="CreepGrowthMediumNydusCreepSourceStart"/>
         <Grown value="Footprint4x4CreepSourceGrown"/>
+        <InfoFlags index="Hidden" value="1"/>
     </CBehaviorCreepSource>
+    <CBehaviorBuff id="OracleStasisTrapTarget">
+        <Modification>
+            <ModifyFlags index="DisableAbils" value="0"/>
+            <ModifyFlags index="SuppressMoving" value="0"/>
+            <ModifyFlags index="SuppressTurning" value="0"/>
+            <AbilClassDisableArray index="CAbilBuild" value="1"/>
+        </Modification>
```

Comment:
Units affected by Stasis can be issued any non-build commands instead of only move commands, to be acted upon stasis expiring

Why are moving and turning not suppressed anymore? Was that required? Can this have side effects?
Can you /dance these units?
Are channeled attacks properly disabled, e.g. beam weapons?